src/components/walls/BrickWall/BrickWall.module.scss

```diff
@@ -0,0 +1,4 @@
+.brick {
```

Comment:
We're using capital letters in CSS too. please. I know this is hard. Consider this for all CSS classes.